.golangci.yml

```diff
@@ -2,13 +2,20 @@ linters-settings:
   funlen:
     lines: 100
     statements: 150
+  goconst:
+    min-occurrences: 5
   gocyclo:
     min-complexity: 20
   golint:
     min-confidence: 0
   govet:
     check-shadowing: true
     enable-all: true
+    disable:
+      - fieldalignment
+  lll:
```

Comment:
Still not sold on line-length restriction being 80 given how awkward it is to wrap some things in Golang.

Reply:
Ah, yeah true, I mean, you know how I feel about it, but this was pretty much a copy/paste job. I'm happy for it to be something wide, specially as I'm personally not touching much code here myself at the moment... lol

Reply:
100 or 120 feels a bit more sensible. I know I try to craft my code for 80 character line lengths most of the time, but when its an awkward line I always get so frustrated having to add a lint ignore.

Reply:
I've set it to `120` now. This also meant I didn't need to modify any existing `*.go` files, which means less risk of any merge conflicts :)